### PR TITLE
DOCSP-33872: Proguard configuration for RN Android

### PR DESCRIPTION
**Pull request by kneth**
## Pull Request Info

Jira ticket: https://jira.mongodb.org/browse/DOCSP-33872

- [PAGE_NAME](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/BRANCH_NAME/): simple Proguard configuration for React Native on Android

### Reminder Checklist

Before merging your PR, make sure to check a few things.

- [ ] Did you tag pages appropriately?
  - genre
  - meta.keywords
  - meta.description
- [ ] Describe your PR's changes in the Release Notes section
- [ ] Create a Jira ticket for related docs-app-services work, if any

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/react-native/install.txt [hunk 1]
```diff
@@ -124,6 +124,16 @@ and add the React Native SDK to it.
             project ``.xcworkspace`` file that you can work with in
             Xcode to run your application.
 
+         .. step:: Extending your Proguard configuration
```
Comment (krollins-mdb) [inline source/sdk/react-native/install.txt:127]:
```suggestion
         .. step:: Extend Android Proguard Configuration
```

File: source/sdk/react-native/install.txt [hunk 2]
```diff
@@ -124,6 +124,16 @@ and add the React Native SDK to it.
...
+
+            For the Android app, you might have to extend the
+            Proguard configuration if Proguard is enabled.
```
Comment (krollins-mdb) [inline source/sdk/react-native/install.txt:130]:
```suggestion
            You may need to extend your Proguard configuration to use
            it with an Android app.
```

File: source/sdk/react-native/install.txt [hunk 3]
```diff
@@ -124,6 +124,16 @@ and add the React Native SDK to it.
...
+            Simply add the following:
```
Comment (krollins-mdb) [inline source/sdk/react-native/install.txt:131]:
```suggestion
            In your Proguard configuration, add the following:
```

**Review by krollins-mdb — COMMENTED** [review]
Thanks, @kneth! I have some style and clarity suggestions. I can handle the updates if you'd like. 😄

**Comment by kneth** [comment]
@krollins-mdb Thank you for the suggestions. Please merge when you are ready :smile:

**Pull request merged**